server/datastore/mysql/app_configs.go

```diff
@@ -47,6 +50,9 @@ func appConfigDB(ctx context.Context, q sqlx.QueryerContext) (*fleet.AppConfig,
 	if err != nil {
 		return nil, ctxerr.Wrap(ctx, err, "unmarshaling config")
 	}
+
+	info.InitializePointers()
```

Comment:
I get this but couldn't understand why we didn't need it before, for other pointers in the config.  I think the reason is the [ApplyDefaultsForNewInstalls](https://github.com/fleetdm/fleet/blob/0354ebdbdb7a865f2b26f23a619a7012d3684d7d/server/fleet/app.go#L904) method -- perhaps instead of this new pattern, we can init UIGitOpsMode in there?

Reply:
I don't think creating defaults would make a huge difference, it would still be nil for all existing deployments. I believe there are a lot of places in the front end where they rely on the struct being populated and I want to try to avoid making them rewrite those areas.

Reply:
That being said, I didn't know about `ApplyDefaultsForNewInstalls` so good callout